src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidationContext.cs

```diff
@@ -46,5 +46,11 @@ public class OpenIdConnectProtocolValidationContext
         /// This id_token is assumed to have audience, issuer, lifetime and signature validated.
         /// </summary>
         public JwtSecurityToken ValidatedIdToken { get; set; }
+
+        /// <summary>
+        /// The id_token is assumed to have audience, issuer, lifetime and signature validated.
+        /// If this property is set, then it will be used instead of the <see cref="ValidatedIdToken"/>.
+        /// </summary>
+        public IJsonWebToken ValidatedIdTokenJwt { get; set; }
```

Comment:
This is the opt in? What if both are set? Maybe a bit more context in the XML on why I would use one over the other would help. @jmprieur what are your thoughts from a PM perspective.

Reply:
@jennyf19 we can control what Asp.net code does so that will be easier. 
We could throw, if both are set as that is ambiguous and most likely a programming error.
We could enforce an order: IJsonWebToken -> JwtSecurityToken

I don't like ambiguity, so i would vote to throw.

Reply:
Would we break developers who are using derived OpenIdConnectProtocolValidator (with virtuals overridden that look only at JwtSecurityToken)?